cypress/integration/seededFlows/adminFlows/users/navigateTabs.spec.js

```diff
@@ -0,0 +1,41 @@
+describe('Navigate User Tabs', () => {
+  describe('As an admin', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/adminUser.json').as('user');
+      cy.get('@user').then((user) => {
+        cy.loginAndVisit(user, '/admin/users/2');
+      });
+    });
+
+    it(`navigates to the "Overview" tab`, () => {
+      cy.findByRole('link', { name: /Notes/i }).should('exist');
+      cy.findByRole('link', { name: /Notes/i }).click();
```

Comment:
(just a suggestion) - if you like you could skip line 12 - unlike some of the other queries, `findByRole` will fail if a matching element isn't found

Also - shouldn't we be clicking the 'Overview' link? I think this test will pass even if on the other tabs because all pages contain `admin/users/2`... maybe instead we could also check

```js
cy.url().should('not.contain', 'tab=')
```

Reply:
> Also - shouldn't we be clicking the 'Overview' link? I think this test will pass even if on the other tabs because all pages contain admin/users/2... maybe instead we could also check

Oops, I should have been more careful when copy-pasta-ing the test below; great catch! 😅  

Also, will do!